tests/framework/installer/installer.go

```diff
@@ -69,7 +70,12 @@ func SkipTestSuite(name string) bool {
 }
 
 func SystemNamespace(namespace string) string {
-	return fmt.Sprintf("%s-system", namespace)
+	if utils.IsPlatformOpenShift() {
+		logger.Infof("For openshift execution used system namespace: %s", namespace)
+		return namespace
+	} else {
```

Comment:
we don't need else we are returning in if statement

Reply:
Done.